src/pkg/storage/testdata/storage.go

```diff
@@ -58,7 +57,7 @@ func NewPrefixedS3Storage(t *testing.T) storage.Storage {
 func GetAndInsertCorso(passphase string) credentials.Corso {
 	// fetch data from flag, env var or func param giving priority to func param
 	// Func param generally will be value fetched from config file using viper.
-	corsoPassph := str.First(flags.CorsoPassphraseFV, os.Getenv(credentials.CorsoPassphrase), passphase)
```

Comment:
This one is a hack just to confirm that its the root case.

Reply:
In case you missed dropping this hack after testing.

Reply:
This confirms that `CorsoPassphraseFV` has some value. Need to check how.

Reply:
I'm confused: is this a TODO/FIXME that needs removal?  Or do we assume the caller is passing FV into the func as the param?  I like the latter better, and it means we wouldn't need to concern ourselves with the FV.  Or, at least, the caller can be in charge of that.

Reply:
flags.CorsoPassphraseFV is to fetch the values from flags. And since this test helper function receives value from envar or config file, flags.CorsoPassphraseFV could be removed. Also flags.CorsoPassphraseFV is a global flag. The value here could be creating conflict.